packages/react/package.json

```diff
@@ -64,9 +64,11 @@
     "@radix-ui/react-dropdown-menu": "0.1.6",
     "@radix-ui/react-slider": "0.1.4",
     "@radix-ui/react-tabs": "0.1.5",
+    "@types/dompurify": "2.3.3",
```

Comment:
This import fixed the following error: https://github.com/rollup/rollup-plugin-commonjs/issues/361

Reply:
This should be in `devDependencies` since it's not a runtime dependency, just a build-time dependency.